ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -11,7 +11,7 @@ import spray.json._
 import scala.util.Try
 
 /** All the JWT payloads that can be used with the JWT auth service. */
-sealed trait AuthServiceJWTPayload
+sealed abstract class AuthServiceJWTPayload extends Product with Serializable
```

Comment:
Since we don't own this code I think it would be better make @mziolekda's team aware of this change (with [this](https://typelevel.org/blog/2018/05/09/product-with-serializable.html) being one of the reasons for this change).